### PR TITLE
ADD missing ARO mappings to manual curation

### PR DESCRIPTION
**Pull request by Vedanth-Ramji**
Additions:
- argannot_curation: (Phe)cpt_strepv:U09991:AAB36569:1412-1948:537 -> ARO: 3000249
- megares_curation: MEG_2114 -> ARO: 3000249, MEG_2430 -> ARO: 3000016, MEG_985 -> ARO: 3000229, MEG_2865 -> ARO:3000387, MEG_7974 -> ARO:3000076
- sarg_curation: AM180355.1.gene2260.p01 -> ARO: 3000250
- resfinder_curation: dldHA2X_1_AL939117 -> ARO:3003970, grdA_1_QJX10702 -> ARO:3007382, EstDL136_1_JN242251 -> ARO:3000387
- resfinderfg_curation: UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase|KF629588.1|pediatric_fecal_sample|CYC -> ARO:3003970
- deeparg_curation: gi:447201629:ref:WP_001278885.1:|FEATURES|cob(I)alamin_adenolsyltransferase|unclassified|cob(I)alamin_adenolsyltransferase -> ARO:0010004

### CODE REVIEW

File: tests/test_lib.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def test_map_to_aro():
         ARO.get_term('ARO:3004623'),
         ARO.get_term('ARO:3000249'),
         ARO.get_term('ARO:3000318'),
-        None
+        ARO.get_term('ARO:3000249')
```
Comment (luispedro) [inline tests/test_lib.py:20]: Can you add an example where `None` is returned?

**Review by luispedro — CHANGES_REQUESTED** [review]
Besides the comment about the tests, can you also add this to the changelog?

**Comment by luispedro** [comment]
Right now, the resfinder table has a column for `Description`, but none of the other ones do. Ideally, this would be more common, but maybe we can start adding descriptions for the new entries going forward?

**Comment by luispedro** [comment]
Looks good. Can you rebase on the current `main`?

**Comment by Vedanth-Ramji** [comment]
I have gone ahead and rebased the commits. It should be ready to merge now

File: integration_tests/run_argnorm.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ def run_cli_test(tool, file, folder, db=None):
 
     output = pd.read_csv(f'integration_tests/outputs/{folder}/{file}', sep='\t')
     golden_file = pd.read_csv(f'outputs/{folder}/{file}', sep='\t')
-    pd.testing.assert_frame_equal(output, golden_file)
+    # pd.testing.assert_frame_equal(output, golden_file)
```
Comment (luispedro) [inline integration_tests/run_argnorm.py:26]: What is this about?

**Review by luispedro — CHANGES_REQUESTED** [review]

File: integration_tests/run_argnorm.py (hunk 2 above)
Comment (Vedanth-Ramji) [reply integration_tests/run_argnorm.py]: Sorry, this was a mistake. I have removed it now.

**Review by luispedro — APPROVED** [review]

**Pull request merged**